### ISSUE TITLE
Improve UI of the Website

### ISSUE THREAD

**Issue by opticod** [NJACKWinterOfCode/E-voting-portal#1]
Currently, the website needs improvement in UI. We need to add improve it and make more materialistic.

For material design guidelines, you can refer:
https://material.io/guidelines/

Comment and consult your proposed ideas and then move on to creating PRs.

**Comment by bksahu** [comment]
Dibs !!! How should I get started ?  Do you want me implement the User Interface Mocks with material design ? 😄

**Comment by opticod** [comment]
Yes, that will be the correct way to get started. Design a user interface mock, you can start by designing only for home page. Then you can move on to coding it. 

If you feel, you can also break down the task into multiple PRs for home page into 

- Toolbar
- Footer
- Main Content

**Comment by bksahu** [comment]
Okay sure. Am I good to go with Materialize.css and Bootstrap ?

**Comment by opticod** [comment]
Materialize.css will be good to go.

**Comment by vanshsoni** [comment]
@opticod 
![screenshot 146](https://user-images.githubusercontent.com/20061884/34481752-e60da70c-efd9-11e7-942d-c41533221433.png)
UI improvment for contact page

**Comment by opticod** [comment]
@vanshsoni Looks good

**Comment by vanshsoni** [comment]
Register Form UI Please Check it then I will make PR
![screenshot 149](https://user-images.githubusercontent.com/20061884/34618501-f4e7a9c0-f264-11e7-944f-0383bb64976d.png)

**Comment by opticod** [comment]
@vanshsoni the bar below Register looks little odd, Try something different for it, rest UI looks fine. 

You can try filling or something else the top section.
